src/AbstractSQLCompiler.ts

```diff
@@ -576,6 +582,8 @@ const compileSchema = (
 	engine: Engines,
 	ifNotExists: boolean,
 ): SqlModel => {
+	abstractSqlModel = optimizeSchema(abstractSqlModel, false);
```

Comment:
If you can chuck this in a separate PR that'd be great

Reply:
https://github.com/balena-io-modules/abstract-sql-compiler/pull/190